manifests/rules/out/chrony.pp

```diff
@@ -1,7 +1,28 @@
-# manage out chrony
-class nftables::rules::out::chrony {
-  nftables::rule {
-    'default_out-chrony':
-      content => 'udp dport 123 accept',
+# @summary manage out chrony
+# @param servers single IP-Address or array of IP-addresses from NTP servers
+class nftables::rules::out::chrony (
+  Variant[Stdlib::IP::Address,Array[Stdlib::IP::Address]] $servers = [],
```

Comment:
```suggestion
  Optional[Variant[Stdlib::IP::Address,Array[Stdlib::IP::Address]]] $servers = undef,
```
Feel it was you who encouraged me down this path generally.

Would need changes below of course.

Reply:
lol, I was just thinking about making it simpler:

```suggestion
  Array[Stdlib::IP::Address] $servers = [],
```

No strong opinion about one way or another, I trend to use the simplest interface to have the less code and the less bugs and maintainability issues, but it imply more constraints for the end user of the module.

Reply:
I think the pure array implementation is cleaner. The idea with `Variant..` is nice if the param was initially a string but we need to support Arrays without a breaking change. Since this parameter is new, I think marking it as array is fine.